app/routes/_gcn.synonyms/synonyms.server.ts

```diff
@@ -110,7 +110,7 @@ export async function searchSynonymsByEventId({
  * BatchWriteItem has a limit of 25 items, so the user may not add more than
  * 25 synonyms at a time.
  */
-export async function createSynonyms(...synonymousEventIds: string[]) {
+export async function createSynonyms(synonymousEventIds: string[]) {
```

Comment:
Why did you change the arguments back?

Reply:
becuse I want to pass in an array of string eventIds [like this](https://github.com/nasa-gcn/gcn.nasa.gov/pull/2007/files#diff-0c3c91f87196d09d921467bd436eb5cfc88043d719ed1ffef34ea277148c875cR50)
```
const synonymousEventIds = ['eventId1', 'eventId2']
const result = await createSynonyms(synonymousEventIds)
```
I do NOT want to pass in eventIds like this:
```
createSynonyms('eventId1', 'eventId2', 'eventId3')
```
unless there is a reason for doing it that way

Reply:
Well, you can call it like this:

```js
createSynonyms(...synonymousEventIds)
```

It doesn't really matter, as long as it's consistent across similar functions in the backend.